### PR TITLE
Fix gemfile to use local gems

### PR DESCRIPTION
**Pull request by seuros**

### CODE REVIEW

**Comment by seuros** [comment]
No, you can't when you have a gem being fetched from a VCS.

You have to match the branches by adding `branch` to the gemfile. 

Testing with the env variable and again without it is much easier than changing bundle config everytime. wdyt ?

**Comment by fernandes** [comment]
@seuros suggestion: can add `scripts/use_local_gems.sh` that does the bundle config and `scripts/use_github_gems.sh` that undo the bundle config

just evaluate what works better, not the intention to bikeshed here heheh

ps: I'm +1 for a bundle config solution because it's related to developers machine, not a global config... anyway 😉

**Comment by apotonick** [comment]
I'm against `bundle config`... I've tried it a few times and found myself going back to messing around with the `Gemfile` because it's much more explicit. Maybe the main problem here is that we had to work a lot with "edge versions" of our gems, and we're now slowly at a point where things are stable. So, maybe this issue will solve itself?

**Comment by seuros** [comment]
@fernandes it won't work. bundle config require to set the branch.
and when the local folder is checking another branch,  it will fail.

File: Gemfile [hunk 1]
```diff
@@ -3,28 +3,31 @@ source 'https://rubygems.org'
...
+  gem "reform", path: "../reform"
+  gem "reform-rails", path: "../reform-rails"
+else
+  gem "reform"
+  gem "reform-rails"
+  gem "trailblazer-operation", github: "trailblazer/trailblazer-operation"
+  gem "trailblazer-macro", github: "trailblazer/trailblazer-macro"
+  gem "trailblazer-macro-contract", github: "trailblazer/trailblazer-macro-contract"
+  gem "trailblazer-activity", github: "trailblazer/trailblazer-activity"
+end
```
[the remark this answers is not in the document]
Comment (fernandes) [reply Gemfile]:
```ruby
git_source(:github) do |repo_name|
  repo_name = "#{repo_name}/#{repo_name}" unless repo_name.include?("/")
  "https://github.com/#{repo_name}.git"
end
```

and there we fixed it! 👍

File: test/test_helper.rb [hunk 2]
```diff
@@ -3,13 +3,12 @@
 require "trailblazer"
 require "minitest/autorun"
 
-# TODO: convert tests to non-rails.
-require "reform"
-require "reform/form/active_model/validations"
-Reform::Form.class_eval do
-  include Reform::Form::ActiveModel::Validations
+# TODO: convert tests to non-rails.	
+require "reform"
```
Comment (saiqulhaq) [inline test/test_helper.rb]: there are many trailing whitespace here

File: Gemfile (hunk 1 above)
[the remark this answers is not in the document]
Comment (apotonick) [reply Gemfile]: Also, some clients etc. do not allow untrusted SSH connections, so HTTPS the way @n-rodriguez suggests is better.

**Pull request merged**